src/flint/types/acb_poly.pyx

```diff
@@ -197,7 +197,7 @@ cdef class acb_poly(flint_poly):
         return u
 
     def __pos__(self):
-        return self # ?
+        return self
```

Comment:
I think something should be kept in the comment here.

This is not how it works for `arb` and `acb` since `+a` rounds to context precision:
```
In [1]: from flint import *

In [2]: a = arb(10**50)

In [3]: a
Out[3]: 1.00000000000000e+50

In [4]: +a
Out[4]: [1.00000000000000e+50 +/- 3.40e+34]

In [5]: a.man_exp()
Out[5]: (88817841970012523233890533447265625, 50)

In [7]: (+a).mid().man_exp()
Out[7]: (4814824860968089, 114)
```
I assume that the comment here is a reminder that maybe this method should be changed to match by using `acb_poly_set_round`.

Reply:
I have reintroduced the comments